wiki/configuration.md

```diff
@@ -180,13 +180,15 @@ The second property **spring.data.mongodb.database** defines a name for the data
 You can configure the RabbitMQ settings using the rabbitmq.* properties.
 Most of the properties should be familiar but a few may need some further explanation.
 The rabbitmq.domain.id, rabbitmq.component.name, rabbitmq.queue.suffix and rabbitmq.queue.durable
-are used to build the queue name on which Eiffel Intelligence listens for messages.
+are used to build the queue name on which Eiffel Intelligence listens for outside messages.
+Multiple routing keys can be defined for this queue with the rabbitmq.binding.key property by writing
+them one after the other in a comma separated string e.g. routing-key1, routing-key2, ... etc.
 An internal waitlist queue is also created that uses all of the previously mentioned properties but also
 attaches the rabbitmq.waitlist.queue.suffix property at the end of the name.
-This queue does not get the routing key binding from the rabbitmq.binding.key property and is only meant for
+This queue does not get the routing key bindings from the rabbitmq.binding.key property and is only meant for
 sending and consuming messages by Eiffel Intelligence.
 It instead uses the non-configurable routing key "eiffel-intelligence.waitlist".
-The rabbitmq.tls.version property specifies the security protocol and you can find valid names
+The rabbitmq.tlsVersion property specifies the security protocol and you can find valid names
```

Comment:
This property should be tls.version on master branch right?

Reply:
Ah yes, there was a mix up on both branches with property names.